src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,128 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.InvalidInputDataException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String LOGIN_EXISTED_IN_STORAGE = "unknownCactus";
+    private static final String DEFAULT_LOGIN = "user1";
+    private static final String DEFAULT_PASSWORD = "user1";
+    private static final int DEFAULT_AGE = 22;

```

Comment:
```suggestion
    private static final int DEFAULT_AGE = 18;

```
default age should be 18